.azure-pipelines/win32-conda-CI.yml

```diff
@@ -62,6 +62,12 @@ jobs:
       pytest tests --doctest-modules --junitxml=junit/test-results.xml
     displayName: 'pytest'
 
+  - script: |
+      coverage run --include=keras2onnx/** tests/benchmark.py
```

Comment:
keras2onnx/ contains tf2onnx code, we should not consider coverage of them. Currently I am using 
coverage run --include=keras2onnx/ke2onnx/ **,keras2onnx/common/ **,keras2onnx/main.py,keras2onnx/funcbook.py,keras2onnx/parser.py,keras2onnx/subgraph.py,keras2onnx/topology.py,keras2onnx/wrapper.py,keras2onnx/_builtin.py tests/test_layers.py
to do the test.

Reply:
Resolved in latest update.